traits/traits.py

```diff
@@ -117,6 +117,17 @@
 }
 
 
+# This function is needed when unpickling historical pickles (pickles
```

Comment:
I didn't actually mean to push this fix until later, but it might as well stay now.